mrbgems/mruby-time/src/time.c

```diff
@@ -211,16 +211,7 @@ mrb_time_wrap(mrb_state *mrb, struct RClass *tc, struct mrb_time *tm)
   return mrb_obj_value(Data_Wrap_Struct(mrb, tc, &mrb_time_type, tm));
 }
 
-static void
-check_num_exact(mrb_state *mrb, double num)
-{
-  if (isinf(num)) {
-    mrb_raise(mrb, E_FLOATDOMAIN_ERROR, num < 0 ? "-Infinity" : "Infinity");
-  }
-  if (isnan(num)) {
-    mrb_raise(mrb, E_FLOATDOMAIN_ERROR, "NaN");
-  }
-}
+void check_num_exact(mrb_state *mrb, double num);
```

Comment:
- functions with external linkage should be prefixed (`mrb_`)

- is `num` of type `double` (declaration) or `mrb_float` (definition)?